mantle/platform/api/aws/images.go

```diff
@@ -329,10 +330,15 @@ func (a *API) CreateImportRole(bucket string) error {
 }
 
 func (a *API) CreateHVMImage(snapshotID string, diskSizeGiB uint, name string, description string) (string, error) {
+	arch := system.RpmArch()
+	if arch == "aarch64" {
+		arch = "arm64"
+	}
+
 	return a.createImage(&ec2.RegisterImageInput{
 		Name:               aws.String(name),
 		Description:        aws.String(description),
-		Architecture:       aws.String("x86_64"),
+		Architecture:       aws.String(arch),
 		VirtualizationType: aws.String("hvm"),
 		RootDeviceName:     aws.String("/dev/xvda"),
```

Comment:
Is `/dev/xvda` vailid for an aarch64 image?

Reply:
yes xvda is supported in arm.